### ISSUE TITLE
Lecturer

### ISSUE THREAD

**Issue by LukasTraenkle** [MaxTrautwein/HS-Esslingen-G3-WS2324-FIS#6]
The Default View for Lecturer, in which the lecturer sees all of their Appointments.

**Comment by LukasTraenkle** [comment]
![image](https://github.com/MaxTrautwein/HS-Esslingen-G3-WS2324-FIS/assets/146187775/29684302-e13a-4fe7-be5e-5ea145c56789)

**Comment by MaxTrautwein** [comment]
# API Anforderungen

## Get Appointments for Lecturer
**Übergabe an Backend** -> User ID 
**Rückrabe vom Backend** -> JSON nach vorlege
```json
{
    "data": [
      {
        "id": 1,
        "name": "SWB3",
        "date": "Donnerstag 12.Oct",
        "start": "10:00",
        "end": "12:15",
        "canceld": false
      },
      {
        "id": 2,
        "name": "SWB4",
        "date": "Donnerstag 12.Oct",
        "start": "10:00",
        "end": "12:15",
        "canceld": false
      }
    ]
  }

```
Ein array of Object with the following content
`id` Id des Appointments
`name` der Name des Appointments z.b.: `SWB3-SW-Softwartechnik`
`date` Datum als String formatiert -> Bitte direkt im gewünschten format übergeben
`start` The Start Time
`end` The End Time
`canceld` bool Status if the event is currently canceld

## Update Canceled Status 
**Übergabe ans Backend** -> ID des Appointments & neuer Status
**Rückgabewert** -> None

---

# Current Look
![image](https://github.com/MaxTrautwein/HS-Esslingen-G3-WS2324-FIS/assets/46843360/a3a0390e-3fbc-419b-a102-a71a3aba982b)

**Comment by MaxTrautwein** [comment]
#15 TODO

**Comment by MaxTrautwein** [comment]
API's are merged with #23 
The Header is still a TODO
